src/dataflow/src/render/sources.rs

```diff
@@ -814,12 +814,22 @@ where
     K2: Codec + timely::Data,
     V2: Codec + timely::Data,
 {
-    let sealed_stream = stream.conditional_seal(
-        &source_name,
-        bindings_stream,
-        write.clone(),
-        bindings_write.clone(),
-    );
+    let mut seal_handle = MultiWriteHandle::new(&write);
+    seal_handle
+        .add_stream(&bindings_write)
+        .expect("handles known to be from same persist runtime");
+
+    // Create a data dependency between `bindings_stream` and `stream`, so that their frontiers
+    // combine. This makes sure that we seal up both streams to the minimum (or meet) of their
+    // frontiers.
+    //
+    // NOTE: The alternative to this is creating a `seal()` operator that can directly take in
```

Comment:
I think eventually we will want to do that, as my understanding is we would need to create a chain of two data deps (or perhaps we could do it with one big call to concatenate) either way I think having one operator that takes each input might eventually be better. No need to do so now

Reply:
Yes, I have something like this planned in my head for adding debezium support, which comes with a third stream that we persist.